workers/unity/Packages/com.improbable.gdk.buildsystem/Util/BuildSupportChecker.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+public static class BuildSupportChecker
+{
+    private static readonly Dictionary<BuildTarget, string> BuildPlatformSupportModuleDirectoryNames;
+
+    static BuildSupportChecker()
+    {
+        BuildPlatformSupportModuleDirectoryNames = new Dictionary<BuildTarget, string>();
+
+        const string linuxStandaloneSupport = "LinuxStandaloneSupport";
+
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux64] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinuxUniversal] = linuxStandaloneSupport;
+
+#if !UNITY_EDITOR_OSX
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneOSX] = "MacStandaloneSupport";
+#endif
+
+#if !UNITY_EDITOR_WIN
+        const string windowsStandaloneSupport = "WindowsStandaloneSupport";
+
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneWindows] = windowsStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneWindows64] = windowsStandaloneSupport;
+#endif
+    }
+
+    public struct BuildSupportCheckResult
```

Comment:
Opportunity to just return a tuple given there's only two fields in the struct

Reply:
Why introduce the struct at all? It has one field and a bool calculated from that field. The addition of the bool adds no information.

Reply:
Could just return an array of targets without build support.

Reply:
Agreed